.claude/skills/managing-templates/SKILL.md

```
@@ -0,0 +1,292 @@
+
+- Export from ComfyUI via **Save → Export**.
+- Prefer ComfyUI started with `--disable-all-custom-nodes`.
+- Embedded workflow in image/video: <https://comfyui-embedded-workflow-editor.vercel.app/>.
+
+Place `your_template_name.json` in `templates/`.
+
+### Step 2 — Thumbnails
+
+In `templates/`:
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)